src/Service/Mailer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use Yiisoft\Form\FormModelInterface;
+use Yiisoft\Mailer\MailerInterface;
+
+final class Mailer
```

Comment:
Name and namespace does not look correct. It should be specific to contact form as implemented in demo:

https://github.com/yiisoft/yii-demo/blob/master/src/Contact/ContactMailer.php

Also it's a good idea to group it in the same way to show the principle of grouping classes by use-case instead of by type.